packages/templates/templates/cheerio-ts/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
     "extends": "@apify/tsconfig",
     "compilerOptions": {
-        "module": "ES2022",
+        "module": "NodeNext",
+        "moduleResolution": "NodeNext",
```

Comment:
Afaik this is implicit with `module: nodenext`. Also, isnt this breaking?

Reply:
It's not, if we specify module, it'll throw unless we also specify moduleResolution

Its not... _breaking_ per say, although some older modules might throw compile errors when import-ed... But we should be moving away from compiling to ES20XX and compile to Node16/Next anyways (or bring in a bundler that isn't tsc for it)

Reply:
I actually saw the opposite behaviour in mikroorm, but i think it was a ts bug in validation. But i do remember reading in docs that this is actually inferred (and it does work that way on my end, i get dynamic imports in the compiled files by [only specifying module](https://github.com/mikro-orm/mikro-orm/blob/master/tsconfig.build.json#L3))

Reply:
Funky, I tried changing just module in our templates and then ran tsc and it threw an immediate error that moduleResolution must also be set to it 😅

Reply:
Well, programming is sometimes weird :) feel free to merge this, even if it would be duplicating things, its fine